components/myforms/HTMLDownload/index.tsx

```diff
@@ -14,6 +14,7 @@ interface HTMLDownloadProps {
   // submissionID: string;
   responseID: string;
   createdAt: number;
+  securityAttribute: string;
```

Comment:
We have a type for this now 

form-types

```
export type SecurityAttribute = "Unclassified" | "Protected A" | "Protected B";
```

Reply:
Will change